nextflow_schema.json

```
@@ -337,6 +361,8 @@
                 },
                 "umitools_umi_separator": {
                     "type": "string",
+                    "pattern": "^\\S+$",
```

```suggestion
                    "pattern": "^\\S+$",
                    "minLength": 1,
                    "maxLength": 1,
```
This will ensure the string is always 1 character long

---

ahh, makes sense. done